src/components/Icon/Icon.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { Icons } from './IconDefinitions';
+
+export const SolidIcon = styled(Icons)`
+  fill: ${props =>
+    props.color ? `${props.theme.colorPrimary}` : `${props.theme.iconPrimary}`};
+  stroke: transparent;
+`;
+export const OutlineIcon = styled(Icons)`
+  fill: none;
+  stroke: ${props =>
+    props.color ? `${props.theme.colorPrimary}` : `${props.theme.iconPrimary}`};
+  stroke-width: ${props =>
+    props.strokeWidth ? `${props.strokeWidth}px` : '2px'};
+  &#logo {
```

Comment:
For readability, I prefer nested definitions to have a new line between.

I could implement stylelint with --fix in our precommit hook to enforce css styling so we don't need to worrry about it. https://github.com/styled-components/stylelint-config-styled-components

Reply:
👍